api/db/api.db.changelog.yml

```diff
@@ -30,4 +30,15 @@ databaseChangeLog:
                   constraints:
                     nullable: false
             tableName: policy_json
+   - changeSet:
+      id: 2
+      author: fujitsu
+      changes:
+      - update:
+          columns:
+          - column:
+              name: guid
+              type: varchar(50)
+              value: "1"
```

Comment:
@pradyutsarma Since the guids are already populated in policy table so it will be a hassle to populate the same ones in scheduler tables as there are 2 tables. So it is ok to have it set to 1  which is easier. And this will not effect the normal processing  and will be taken care of when the synchronisation happens. Also these are used by the synchroniser only.